### DISCUSSION TITLE
Is there anyway to custom an icon for an unpackaged app?

### DISCUSSION THREAD

**Discussion by sxlllslgh** [microsoft/WindowsAppSDK#4452] [Q&A] [answered]
I mean the icon of the **exe file**, not ones in the title bar or task bar. Thank you very much!

**Comment by mareksm** [comment, marked as answer]
You can add one in rc file as ICON resource: https://learn.microsoft.com/en-us/windows/win32/menurc/icon-resource

**Comment by castorix** [comment]
The Program icon in Project Properties changes the icon of the executable (at least with Windows App SDK 1.4.23) :

![image](https://github.com/microsoft/WindowsAppSDK/assets/22345506/3d0f8959-80f5-41cd-b4cc-9dcf12b15410)
